File: randgenDDMIST.py
from random import *
import sys

file = open('input.txt', 'w')
sys.stdout = file

N = 200
Q = 200


print(N)
for i in range(N):
    print(randint(1,1e9),end= " ")

print(Q)
for i in range(Q):
    L = randint(1,N)
    R = randint(L,N)
    print(L,R)

file.close()

File: randgenDDMIST.cpp
from random import *

M = 100000
N = 2000
D = 5
print(N,D,file = f)

f = open("ddinput.txt", "a")

for i in range(N):
    for j in range(D):
        print(randint(1,M),end = " ")
    print("",file = f)

f.close()